libs/api/domains/vehicles/src/models/usersVehicles.model.ts

```diff
@@ -196,3 +199,60 @@ export class VehiclesList {
   })
   createdTimestamp?: string
 }
+
+@ObjectType()
+export class VehicleListed {
```

Comment:
Weird name, why not just Vehicle?

Reply:
I kinda agree, but there are three different vehicle responses.

1. Detail vehicle old service
2. Overview vehicle old service
3. Overview vehicle new service.

This is nr 3. A type of vehicle only shown in the list. So that's where the name comes from 🤷 
`Vehicle` could be a bit confusing because of that.